wiki/Ranking_Criteria/en.md

```diff
@@ -142,52 +137,62 @@ Custom naming must follow a common theme or pattern related to the song and must
 ## Metadata
 
 ### Rules
-
-- **The following technical limitations or conventions are put in place to ensure consistency:**
-  - **Metadata must be consistent across all difficulties of a beatmap set.** 
-  - **The artists of a song must be traceable to existing people.** As such, programs like Vocaloids or fictional characters cannot be used as the sole artist of any given song.
-  - **Use the metadata of a song as is without adding any additional markers, such as `Full Version` and the likes.** If your track is a mashup or something among those lines, use whatever the artist supplies while paying respect to proper English syntax rules.
-  - **Guest beatmap creators, storyboarders, and hit sounders must be added to the tags of a beatmap set.** This is to give credit where credit is due and helping others identify the main contributors of any given beatmap set.
-  - **You must use the Source field if the song comes from, is remixed from or specifically fan-made for a video game, movie, or series.** Website names are not an acceptable Artist nor Source. 
-  - **If the artist or title field exceeds the uploadable maximum length, or both together cause Windows filenames for the `.osu` files to exceed 255 characters, any additional markers from the fields causing this have to be dropped consistently and if this is still not sufficient, the corresponding fields need to be abbreviated reasonably and end in `...` to signal that this song title has been shortened.**
-  - **Special characters that have both a full-width and half-width equivalent must retain their original printing forms if they are used in the unicode artist fields,** so that searching while copying and pasting the original song title is always possible. This does not apply to additional information that is printed in half-width to begin with because searching for these will be ambiguous no matter what.
+#### Technical
+- **Metadata must be consistent across all difficulties of a beatmap set.**
+- **Guest mappers, storyboarders, skinners and hitsounders must be added to the tags of a beatmap set.** This is to give credit where credit is due and helping others identify the main contributors of any given beatmap set. Usernames containing whitepsaces should replace them with underscores.
+- **Official Sources must be used as references for metadata unless none are available,** in which case use what is most common and recognizable.
```

Comment:
```suggestion
- **Official sources must be used as references for metadata unless none are available,** in which case the most common and recognizable information should be used.
```

Reply:
that makes the sentence more complicated isntead of keeping it simple

Reply:
what you have currently contains incorrect grammar (tenses don't match)